packages/api-generator/.gitignore

```diff
@@ -1 +1,2 @@
 /dist/
+package-lock.json
```

Comment:
lock file was untracked when I installed packages. So added it to .gitignore as I though not tracking lock file is intentional.
If it wasn't I will remove it.

Reply:
Use yarn instead of npm

Reply:
yarn won't install on my linux machine, so I have given up on it. But I will take care of this.

Reply:
We use yarn workspaces, so you kinda need it.

Reply:
Don't worry about it, I will take care of it